ivy/array/general.py

```diff
@@ -657,9 +657,73 @@ def stable_divide(
             a numpy array containing the elements of numerator divided by
             the corresponding element of denominator
 
+        Examples
```

Comment:
Please revert these changes as it's not related to the function being reformatted through your task.